amd_openvx_extensions/amd_rpp/source/tensor/TensorAddTensor.cpp

```diff
@@ -0,0 +1,237 @@
+/*
+Copyright (c) 2024 Advanced Micro Devices, Inc. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#include "internal_publishKernels.h"
+#if _WIN32
+#include <intrin.h>
+#else
+#include <immintrin.h>
+#include <smmintrin.h>
+#include <x86intrin.h>
+#endif
+
+struct TensorAddTensorLocalData {
+    Rpp32u deviceType;
+    Rpp32u nbatchSize;
+    RppPtr_t pSrc1;
+    RppPtr_t pSrc2;
+    RppPtr_t pDst;
+    int *srcDims1;
+    int *srcDims2;
+    RpptROI *pSrcRoi;
+    RpptROI *pDstRoi;
+    size_t inputTensorDims1[RPP_MAX_TENSOR_DIMS];
+    size_t inputTensorDims2[RPP_MAX_TENSOR_DIMS];
+    vx_enum inTensorType;
+    vx_enum outTensorType;
+};
+
+void update_destination_roi(TensorAddTensorLocalData *data, void *roi_tensor_ptr_src, void *roi_tensor_ptr_dst) {
+    data->pDstRoi = (RpptROI *)roi_tensor_ptr_dst;
+    data->pSrcRoi = (RpptROI *)roi_tensor_ptr_src;
+    for (uint i = 0; i < data->nbatchSize; i++) {
+        data->pDstRoi[i].xywhROI.xy.x = data->pSrcRoi[i].xywhROI.xy.x;
+        data->pDstRoi[i].xywhROI.xy.y = data->pSrcRoi[i].xywhROI.xy.y;
+    }
```

Comment:
We need to update the w and h and not x and y

Also do you need to update the ROI in here for this function
Please check this @SundarRajan28